_posts/2016-10-28-Sharding.markdown

```diff
@@ -20,7 +20,16 @@ source "https://github.com/CocoaPods/Old-Specs"
 </code>
 </pre>
 
-To the top of your Podfile, and CocoaPods will only use the archived repo, instead of using the new repo structure.
+To the top of your Podfile, and CocoaPods will only use the archived repo, instead of using the new repo structure. You will also need to set your local Specs repo to a version before the transition:
+
+<pre>
+<code>
+cd ~/.cocoapods/repos/master/
+git checkout predates_sharding_branch
```

Comment:
But when CocoaPods decides to update it, it's going to fail right?

```
 ~/.c/r/master  ⌂ v0.32.1  git pull                                                                                        4236ms  Wed 16 Nov 06:59:17 2016
remote: Counting objects: 1140, done.
remote: Compressing objects: 100% (687/687), done.
remote: Total 1140 (delta 716), reused 817 (delta 398), pack-reused 0
Receiving objects: 100% (1140/1140), 126.74 KiB | 0 bytes/s, done.
Resolving deltas: 100% (716/716), completed with 254 local objects.
From https://github.com/CocoaPods/Specs
   fb2d7a3..9fbe9d9  master     -> origin/master
You are not currently on a branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>
```

This gives a status code of not 0. So then doing a `pod install` would fail?